app/build.gradle

```diff
@@ -17,33 +17,83 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+
+    packagingOptions {

```

Comment:
What are these used for?

Reply:
Right now its to resolve some conflicts when building the app due to some duplicate files between RxJava 1/2. This is only temporary till we establish which one we want to use.